src/groovy/com/unifina/signalpath/messaging/EmailModule.groovy

```diff
@@ -70,18 +71,36 @@ class EmailModule extends ModuleWithSideEffects {
 			}
 		}
 
-		// Create body for the email
-		String messageBody = ("\n" +
-				"Message:\n" +
-				"${message.getValue()}\n\n" +
-				"Event Timestamp:\n" +
-				"${df.format(globals.time)}\n\n" +
-				"Input Values:\n" +
-				"${inputValues}\n" +
-				(lastEmailBlocked ? "\nWARNING: Some emails between this and the last mail have not been sent because of trying to send too frequently.\n" : "") +
-				"")
-
-		return messageBody
+		String body = """
+Message:
```

Comment:
The template needs a bit of additional wording/instruction. How about this:

```
This email was sent by one of your running Canvases on Streamr.

Message:
{0}

Event Timestamp:
{1,date,yyyy-MM-dd HH:mm:ss.SSS}

Input Values:
{2}

To view, edit, or stop the Canvas that sent this message, click the below link:
{3}

{4}
```

Reply:
@kare how about this template change?

Reply:
Fixed